drivers/modem/quectel-bg9x.c

```diff
@@ -953,6 +953,64 @@ static const struct setup_cmd setup_cmds[] = {
 	SETUP_CMD_NOHANDLE("AT+QICSGP=1,1,\"" MDM_APN "\",\"" MDM_USERNAME "\", \"" MDM_PASSWORD "\",1"),
 };
 
+
+/* Func: modem_pdp_context_open
+ * Desc: This helper function is called from modem_setup, and is
+ * used to open the PDP context. If there is trouble opening the
+ *  PDP context, we try to deactive and reactive MDM_PDP_ACT_RETRY_COUNT times.
+ *  If it fails, we return an error.
+ */
+
+
+static int modem_pdp_context_open(const int contextID)
+{
+
+	int ret;
+	int pdp_act_retry_count = 0;
```

Comment:
This function is all about activating PDP context, so I would remove `pdp_act_` prefix. Seems obvious what the retry count is about.